clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/api/ApplicationService.java

```diff
@@ -78,19 +77,6 @@
   @DELETE("/v2/apps/{guid}/instances/{index}")
   Call<ResponseBody> deleteAppInstance(@Path("guid") String guid, @Path("index") String index);
 
-  @POST("/v3/processes/{guid}/actions/scale")
-  Call<ResponseBody> scaleApplication(
-      @Path("guid") String guid, @Body ScaleApplication scaleApplication);
-
-  @PATCH("/v3/processes/{guid}")
-  Call<Process> updateProcess(@Path("guid") String guid, @Body UpdateProcess updateProcess);
-
-  @GET("/v3/processes/{guid}")
-  Call<Process> findProcessById(@Path("guid") String guid);
-
-  @GET("/v3/processes/{guid}/stats")
-  Call<ProcessResources> findProcessStatsById(@Path("guid") String guid);
-
```

Comment:
Is the plan to move other methods to different interfaces (like one interface for routes, one for packages, etc)?

Reply:
I think as they get big enough it makes sense. Overall I think our nomenclature should be as similar to the cloud foundry api as possible. Another example, I renamed "scaleApplication" to "scaleProcess" since thats actually whats happening (v3/processes/:guid/actions/scale). That could cause confusion for someone looking at it for the first time if they don't have the context